sphinx_gallery/gen_gallery.py

```diff
@@ -294,7 +294,8 @@ def setup(app):
 
     app.add_stylesheet('gallery.css')
 
-    if 'sphinx.ext.autodoc' in app._extensions:
+    extensions_attr = '_extensions' if hasattr(app, '_extensions') else 'extensions'
```

Comment:
It wants the value corresponding to either the `'_extensions'` attribute or the `'extensions'` attribute, depending on which exists.